inc/gravity-forms.php

```diff
@@ -115,4 +118,11 @@
     }
 
     return $field_content;
-}, 10, 2 );
\ No newline at end of file
+}, 10, 2 );
+
+// Extend expiration of save and continue links from 30 days to 1 year
+add_filter( 'gform_incomplete_submissions_expiration_days', 'gwp_days', 1, 10 );
```

Comment:
If I am not wrong that should be ```add_filter( 'gform_incomplete_submissions_expiration_days', 'gwp_days', 10, 1);``` instead see: https://developer.wordpress.org/reference/functions/add_filter/

Reply:
Thanks @marcinca I've corrected this. Could you review again please?